[
   {
      "author_association" : "NONE",
      "body" : "Can somebody elaborate on how to produce the error. It seems the coverage report is still generated fine even given individual functional tests failed",
      "created_at" : "2019-04-08T03:20:22Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/15648#issuecomment-480669693",
      "id" : 480669693,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/15648",
      "node_id" : "MDEyOklzc3VlQ29tbWVudDQ4MDY2OTY5Mw==",
      "updated_at" : "2019-04-08T03:20:22Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/480669693",
      "user" : {
         "avatar_url" : "https://avatars0.githubusercontent.com/u/3409608?v=4",
         "events_url" : "https://api.github.com/users/crackercracked/events{/privacy}",
         "followers_url" : "https://api.github.com/users/crackercracked/followers",
         "following_url" : "https://api.github.com/users/crackercracked/following{/other_user}",
         "gists_url" : "https://api.github.com/users/crackercracked/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/crackercracked",
         "id" : 3409608,
         "login" : "crackercracked",
         "node_id" : "MDQ6VXNlcjM0MDk2MDg=",
         "organizations_url" : "https://api.github.com/users/crackercracked/orgs",
         "received_events_url" : "https://api.github.com/users/crackercracked/received_events",
         "repos_url" : "https://api.github.com/users/crackercracked/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/crackercracked/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/crackercracked/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/crackercracked"
      }
   }
]
